code/game/objects/effects/landmarks.dm

```diff
@@ -97,28 +97,6 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/awaystart) //Without this away mission
 	spawner_list = GLOB.latejoin
 	return ..()
 
-/obj/effect/landmark/spawner/late/cryo
-	name = "Late Join Cryo"
-
-/obj/effect/landmark/spawner/late/cryo/Initialize(mapload)
-	spawner_list = GLOB.latejoin_cryo
```

Comment:
Make sure you also remove these lists if appropriate, and any associated code for spawning.

Reply:
The "associated code for spawning" is set up to choose whatever is picked in your preferences, which, the default is the arrivals shuttle, and there's no method whatsoever to change that so far as I'm aware. The code for spawning is, from what I can tell, VERY generous in what it allows, to the point where this won't have any effect so long as nobody has these set in their preferences, which they shouldn't anyways, and is an issue for other reasons anyways.

Reply:
If all additional spawns are being removed, the code should just put you in the shuttle. No need for code that checks your prefs

Reply:
That's a bit more work but I'll see if I can do it without too much stuff falling apart.